fit/core-reference/src/test/java/org/apache/syncope/fit/AbstractITCase.java

```diff
@@ -702,4 +767,9 @@ protected void removeLdapRemoteObject(
         } while (results.isEmpty() && i < maxWaitSeconds);
         return results;
     }
+
+    protected void cleanUpRemediations() {
```

Comment:
This method seems to be used only within `PullTaskITCase`: can you move it to that class?